src/jsfiles/caesar.js

```diff
@@ -0,0 +1,14 @@
+const caesar = (str) => {
```

Comment:
You are missing the shift parameter. Remember Caesar Cipher is not only for shift of 1 but can be any. This technically not a full caesar cipher. 
Reread here: http://practicalcryptography.com/ciphers/caesar-cipher/
Implement changes accordingly and update tests accordingly.